### DISCUSSION TITLE
关于跨网盘复制

### DISCUSSION THREAD

**Discussion by heguhls** [AlistGo/alist#4397] [General]
想问下 alist跨网盘复制时产生的临时文件都在哪里啊 这个路径是可以设置的吗

**Comment by ririye** [comment]
![QQ截图20230516142418](https://github.com/alist-org/alist/assets/124979523/7dcac146-5143-434d-8d2b-7b43ff39d368)
写在你alist所在目录下的data文件夹里的config.json文件里，用记事本或者notepad++等工具打开就看到了，想改到那里就把原来的替换掉就可以啦

**Reply by heguhls** [reply]
感谢大佬。我是之前跨网盘转了一些大文件，结果产生的临时文件太多、太大把盘撑满了，后面的传输全失败了，所以想想办法改改这个。现在离上次转文件的时间有点久了 忘记那些临时文件的格式和文件名是什么了。对这些文件的话我应该就是改"temp_dir"吗

**Reply by ririye** [reply]
对，是滴，改在别的盘的话记得改成绝对路径，还有\一定要2个,或者你直接把alist文件和它同级目录的data文件夹一起转移到别处也行

**Reply by heguhls** [reply]
谢啦
